code/__HELPERS/game.dm

```diff
@@ -75,17 +75,13 @@
 		max(list_y))
 
 // Like view but bypasses luminosity check
-
 /proc/get_hear(range, atom/source)
-
 	var/lum = source.luminosity
 	source.luminosity = 6
 
-	var/list/heard = view(range, source)
+	. = view(range, source)
```

Comment:
How hot is this proc? is it worth this level of micro op?

Reply:
Camera code uses this frequently and its a minor change